apps/site/assets/ts/__v3api.d.ts

```diff
@@ -73,16 +73,19 @@ export interface Prediction {
   track: string | null;
 }
 
-export interface Route {
-  alert_count: number;
+export interface BareRoute {
```

Comment:
I think this should be the other way around -- `Route` should be the base interface and it should get extended and renamed when necessary.

Reply:
This would touch a lot of code and I wanted to avoid touching unnecessary files. I can open another PR after to fix this.

If I had opened all my fixups as individual PRs it would have been a lot.